events/iisc_pulse_events.txt

```diff
@@ -113,8 +113,14 @@ country_event = {
 	id = iisc_pulse.20
 	is_triggered_only = yes
 	hide_window = yes
+	trigger = {
+		NOR = {
+			has_global_flag = iisc_paused
+			has_global_flag = iisc_stopped
+		}
+	}
 	immediate = {
-		if = {
+		every_country = {
```

Comment:
I'm not sure what's going on here, this might have to do with the change I merged in.